test/helpers.go

```diff
@@ -88,7 +88,7 @@ func RandomReader(seed, size int) *bytes.Reader {
 
 // SetupTarTestFixture extracts the tarFile to outputDir.
 func SetupTarTestFixture(t testing.TB, outputDir, tarFile string) {
-	err := System("sh", "-c", `(cd "$1" && tar xz) < "$2"`,
+	err := System("sh", "-c", `(cd "$1" && tar xzf - ) < "$2"`,
```

Comment:
On OpenBSD and FreeBSD, `tar` doesn't read from stdin by default (which it does on Linux and darwin), but from `/dev/sa0`. This instructs tar to read from stdin with `f -`.